core/src/main/java/com/google/errorprone/bugpatterns/MissingOverride.java

```diff
@@ -46,6 +47,20 @@
 )
 public class MissingOverride extends BugChecker implements MethodTreeMatcher {
 
+  /**
+   * if true, don't warn on missing @Override annotations inside interfaces
+   */
+  private final boolean ignoreInterfaceOverrides;
+
+  public MissingOverride() {
+    this.ignoreInterfaceOverrides = false;
```

Comment:
You could keep the default (i.e., `false`) in one place by replacing this constructor implementation with `this(ErrorProneFlags.empty());`.